baselines/train/configs.py

```diff
@@ -23,13 +23,32 @@
     'territory__rooms_1',
     'territory__rooms_2',
     'territory__rooms_3',
+    'daycare_0',
+    'commons_harvest__partnership_0',
+    'commons_harvest__partnership_1',
+    'commons_harvest__partnership_2',
+    'commons_harvest__partnership_3',
+    'commons_harvest__partnership_4',
+    'commons_harvest__partnership_5',
+    'commons_harvest__open_0',
+    'commons_harvest__open_1',
+    'commons_harvest__closed_0',
+    'commons_harvest__closed_1',
+    'commons_harvest__closed_2',
+    'commons_harvest__closed_3',
+    'commons_harvest__private_property_pc_0',
+    'commons_harvest__private_property_pc_1',
+    'commons_harvest__private_property_0',
+    'commons_harvest__private_property_1',
+    'commons_harvest__open_disable_zapping_0',
+    'commons_harvest__open_disable_zapping_1',
+    'commons_harvest__open_abundance'
```

Comment:
We add two scenarios for evaluation minimum per substrate
```suggestion
    'commons_harvest__open_abundance_0',
    'commons_harvest__open_abundance_1',
```